java/compat/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
```gradlew```, ```gradlew.bat```, and the jars in ```gradle/``` are meant to allow you to check out and build without installing the build tool globally. It's common in gradle projects, but I'm not a huge fan. I'd like to investigate getting rid of them in the near future in favor of assuming you have gradle installed, but I have some questions about how that interacts with IDEA that I need to answer first.

I initially symlinked these files to the ones in core, but I couldn't get git on Windows to handle that properly. I've got a branch going on the side to try to have a single gradle entry point for all of Java, but I think that may depend on getting the gbc hook working.

In short, I would like to accept the redundancy for the moment.